### ISSUE TITLE
no email about assembly failing

### ISSUE THREAD

**Issue by ARWattam** [PATRIC3/patric3_website#141]
also, no hint as to what I can do to correct on the next submission

**Comment by dmachi** [comment]
We do have the ability to view some errors now.  I implemented emailing capabilities for users.  Bob is going to add the notification from the Job side.  Not until after R2.

**Comment by olsonanl** [comment]
I'm looking into this; I've created a new github user patric-app-service and have figured out the tech to allow that users to post issues in an issue tracker. Current idea is to have application failures post to a tracker that is dedicated to the app service. We'll see how it goes.

**Comment by dmachi** [comment]
Sounds good. I believe this ticket was primarily about emailing to the users (via p3-user) upon job completion, are you going to do both at the same time?

**Comment by olsonanl** [comment]
Yup, that's the plan. I got to thinking about the tickets since I want us to be notified as well, but don't have a way to mail to admins and the tracker makes more sense to me for that.

**Comment by olsonanl** [comment]
I just tested out the user notification API and it worked fine for me.

**Comment by hyoo** [comment]
@olsonanl are we still planning to do this?

**Comment by olsonanl** [comment]
Yes, I would like to.

**Comment by mshukla1** [comment]
@olsonanl From the previous comments, it seems you had a working prototype of job status notification working in 2015. Is it correct? 

Is this something that can be rolled out with the release of the new scheduler in May?

**Comment by mshukla1** [comment]
@olsonanl let me know if you like to leave job status notification for the P4. In that case, I can make a note of it elsewhere and close this ticket.

**Comment by olsonanl** [comment]
No, this should be part of the new scheduler.
